README.md

```diff
@@ -463,10 +520,11 @@ style.xml
 ```
 
 ## Authorizing payment
-Some payment methods require the customer to authorize the payment using an authorization URL. This includes [3-D Secure verification](https://docs.opn.ooo/fraud-protection#3-d-secure), [Internet Banking payment](https://docs.opn.ooo/internet-banking), [Alipay](https://docs.opn.ooo/alipay), etc. Opn Payments Android SDK provides a built in class to handle the authorization.
 
-On payment methods that require opening the external app (e.g. mobile banking app) to authorize the transaction, set the *return_uri* to a **deeplink** or **applink** to be able to open the merchant app. Else, after the card holder completes authorizing the transaction on the external app, the flow redirects to the normal link in the *return_uri* and opens it on the browser app, and therefore results in the payment not being completed.
+Some payment methods require the customer to authorize the payment using an authorization URL. This includes [3-D Secure verification](https://docs.opn.ooo/fraud-protection#3-d-secure), [Internet Banking payment](https://docs.opn.ooo/internet-banking), [Alipay](https://docs.opn.ooo/alipay), etc. Opn Payments Android SDK provides a built in class to handle the authorization.
 
+On payment methods that require opening the external app (e.g. mobile banking app) to authorize the transaction, set the _return_uri_ to a **deeplink** or **applink** to be able to open the merchant app. Else, after the card holder completes authorizing the transaction on the external app, the flow redirects to the normal link in the _return_uri_ and opens it on the browser app, and therefore results in the payment not being completed.
```

Comment:
I think instead of `Else, after the....` we better use `Otherwise after card holder completes`

Reply:
And could you please explain to me why payment will not be completed if I open return_uri in a browser?

Reply:
@annatotest  The previous explanation was like this but i believe that the actual payment is completed but the callback to the app will not be completed as the browser will stay opened in the merchant's app so in order for the browser to close automatically the merchant needs to implement the described set up.